utils/RequestUtil.js

```diff
@@ -73,20 +73,23 @@ var RequestUtil = {
   json: function (options) {
     // Default assign options to empty object
     options || (options = {});
+    var requestID = options.url;
```

Comment:
This isn't part of this PR, but we should actually create a better unique id. Sometimes there's different query parameters that we pass or headers. Could you think of a good way to create a unique id out of all of this?
Also, keep in mind that `_timestamp` needs to be ommitted from the params.